oidb/pepper/service/src/lib.rs

```diff
@@ -0,0 +1,185 @@
+use crate::vuf_keys::VUF_SCHEME0_SK;
+use anyhow::{anyhow, bail, ensure};
+use aptos_oidb_pepper_common::{
+    jwt::Claims, nonce_derivation, nonce_derivation::NonceDerivationScheme,
+    pepper_pre_image_derivation, pepper_pre_image_derivation::PepperPreImageDerivation, vuf,
+    vuf::VUF, PepperRequest, SimplePepperRequest,
+};
+use jsonwebtoken::{Algorithm::RS256, Validation};
+use once_cell::sync::Lazy;
+use rand::thread_rng;
+use std::collections::HashSet;
+
+pub mod about;
+pub mod jwk;
+pub mod vuf_keys;
+
+pub type Issuer = String;
+pub type KeyID = String;
+
+/// The core processing logic of this pepper service.
+pub async fn process(request: PepperRequest) -> anyhow::Result<String> {
+    let mut rng = thread_rng();
+    let mut aead_rng = aes_gcm::aead::OsRng;
```

Comment:
why are we using two different RNGs?

Reply:
the exact aead library version I currently use is unhappy with the the 1st rng. And I haven't dived too deep to address it.

But yeah that's annoying, and i will add a todo here...